script/run_test.py

```diff
@@ -37,6 +37,7 @@ def get_target(debug, postfix=""):
         clean_bazel(args.docker)
     
     if args.android:
```

Comment:
Why should we wipe the base dir?

Reply:
I updated that line because when we run test many times, build is failed because the built file already exists.
To solve this issue, I think several ways are possible:
- remove the build file all the times
- change this function usage with exist_ok=True: https://github.com/mrsnu/band/blob/e265b398aecda93294d37ebbae3d06f4ffa98794/script/utils/util.py#L66

However, with second way, as the `shutil.copytree` doesn't support overwrite, I think using `disutils.dir_util.copy_tree` is needed for multiple build test.
How's your opinion about that?

Reply:
FYI: I left the logs 
```
Traceback (most recent call last):
  File "script/run_test.py", line 55, in <module>
    copy(f'bazel-bin/band/test', get_target(args.debug))
  File "/workspaces/band/script/utils/util.py", line 66, in copy
    shutil.copytree(src, dst, ignore_dangling_symlinks=True, symlinks=True)
  File "/usr/lib/python3.8/shutil.py", line 557, in copytree
    return _copytree(entries=entries, src=src, dst=dst, symlinks=symlinks,
  File "/usr/lib/python3.8/shutil.py", line 458, in _copytree
    os.makedirs(dst, exist_ok=dirs_exist_ok)
  File "/usr/lib/python3.8/os.py", line 223, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: 'test_bin/armv8-a/release/test'
```

Reply:
I'll separately handle this issue in another PR.